### PR TITLE
[Evaluators][TA] Fix get candle manager call

### PR DESCRIPTION
**Pull request by Herklos**

### CODE REVIEW

**Comment by GuillaumeDSM** [comment]
I did it locally and forgot to push it 👎

**Comment by Herklos** [comment]
> I did it locally and forgot to push it

Classic one :smile:

File: Evaluator/TA/momentum_evaluator.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ def __init__(self):
 
     async def ohlcv_callback(self, exchange: str, exchange_id: str, symbol: str,  time_frame, candle):
         period_length = 14
-        candle_data = self.get_candle_manager(exchange, exchange_id, symbol, time_frame).get_symbol_close_candles().base
+        candle_data = self.get_symbol_candles(exchange, exchange_id, symbol, time_frame).get_symbol_close_candles()
```
Comment (GuillaumeDSM) [inline Evaluator/TA/momentum_evaluator.py]: We still need the ````.base```: get_symbol_close_candles is not returning ```.base```

Comment (Herklos) [reply Evaluator/TA/momentum_evaluator.py]: Yes but it is working, I don't really understand why.

The `.base` is returning None and without `.base` a numpy array is returned.

**Pull request merged**